openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -82,6 +84,31 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
         }
     }
 
+    /**
+     * Checks if the current operative system is Alpine.
+     *
+     * @return true when the operative system is Alpine and false otherwise.
+     */
+    static private boolean isAlpine() {
+        final String osReleasePath = "/etc/os-release";
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(osReleasePath));
+
+            String line;
+            while ((line = reader.readLine()) != null) {
+                if (line.equals("ID=alpine")) {
+                    reader.close();
+                    return true;
+                }
+            }
+        } catch (IOException ex) {
+            return false;
```

Comment:
The exception must not be swallowed. Please log the exception and throw the exception. We can't assume that we aren't running in Alpine if this throws an exception or am I missing something?

Reply:
This code was removed.